lua/entities/gmod_wire_expression2/core/hologram.lua

```diff
@@ -1003,13 +1003,27 @@ e2function void holoPos(index, vector position)
 	WireLib.setPos(Holo.ent, Vector(position[1],position[2],position[3]))
 end
 
+e2function vector holoPos(index)
+	local Holo = CheckIndex(self, index)
+	if not Holo then return end
```

Comment:
Missing return here

Reply:
Your analysis is greatly appreciated. Maybe for future reference you should make a note in the style guide or somewhere else that attribute should be used whenever possible.